src/schema/basic_classes.yaml

```
@@ -489,6 +494,11 @@ slots:
     range: DataGeneration
     mappings:
       - prov:wasInformedBy
+    structured_aliases:
+      was_informed_by:
+        contexts:
+        -  https://bitbucket.org/berkeleylab/jgi-jat
+        predicate: exact_synonym
```

Did you mean for this to be uppercase? I am seeing an error about `exact_synonym` in the GitHub Actions logs.

```suggestion
        predicate: EXACT_SYNONYM
```